sys/config/include/config/config.h

```diff
@@ -188,6 +188,15 @@ int conf_register(struct conf_handler *cf);
  */
 int conf_load(void);
 
+/**
+ * Load configuration from a specific registered persistence source.
+ * Handlers will be called for configuration subtrees for
+ * encountered values.
+ *
+ * @return 0 on success, non-zero on failure.
+ */
```

Comment:
Comment copied from function above, should be updated to describe this function.

Also, note the existence of conf_get_stored_value(), which also can be used in finding a value associated with a specific name.

Reply:
Yeah, that could be used but we wanted the callbacks getting called as well. Hence, the additional API.